### DISCUSSION TITLE
Typescript references

### DISCUSSION THREAD

**Discussion by rlsgit** [adonisjs/core#3578] [Help]
Trying to use references in tsconfig.json for local library support...
tsc works fine with "--build" option...
node ace build does not. 

Is it possible to get the --build option included in node ace build? Or am I missing something else that would work. What else is node ace build doing that needs to happen alongside tsc?

**Comment by guntribam** [comment]
please share logs, errors, etc

**Comment by phatmann** [comment]
Without the --build option, the dependent typescript projects won't be built. So you will get a type error for each reference to a file in the dependent project.
